lib/filesystem/include/irods/filesystem/path_utilities.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef IRODS_FILESYSTEM_PATH_UTILITIES_HPP
+#define IRODS_FILESYSTEM_PATH_UTILITIES_HPP
+
+#include "irods/filesystem/path.hpp"
+#include "irods/filesystem/filesystem_error.hpp"
+
+#include "irods/rodsDef.h"
+#include "irods/rodsErrorTable.h"
+#include "irods/system_error.hpp"
+
+#include <cstring>
+
+namespace irods::experimental::filesystem
+{
+    /// Returns whether the path element \p _c represents the preferred path separator.
+    ///
+    /// 4.2.12
```

Comment:
Should this be `\since 4.2.12`, or does Doxygen pick this up?

Reply:
That's a typo. Will fix

Reply:
Done